test/clients.go

```diff
@@ -71,7 +72,7 @@ func BuildClientConfig(kubeConfigPath string, clusterName string) (*rest.Config,
 
 // UpdateConfigMap updates the config map for specified @name with values
 func (client *KubeClient) UpdateConfigMap(name string, configName string, values map[string]string) error {
```

Comment:
Seems like now would be the time to break these APIs as well, but I'm not sure where they are used.  Eventing, perhaps?

Reply:
done in efdb1fc3a7cff6eec4d060e3a5744446051894b5